### ISSUE TITLE
Feature: please include support for multiple sorting variables to the plot_composition()

### ISSUE THREAD

**Issue by wolass** [microbiome/microbiome#5]
Hi, 
I just wanted to highlight that there should be an option to sort the samples according to more than one factor. 

Lets assume that I have two variables "disease" and "lesion". I would like my samples to be sorted now according to both of these variables meaning first I want the lesional samples to be grouped and there can be two groups Lesional or non-lesional, then I want the non-lesional to be grouped again into two subcategories: "disease+" and "disease-". 

As it is imposible for a "disease-" sample to fall into the lesional category it will work perfectly. 

Hope this clarifies what I mean. 

Best,
W

**Comment by antagomir** [comment]
I agree this would be nice to have in principle. There is already an option to provide sample ordering manually via the sample.sort input argument, and finding the sorting in the way you are suggesting would be a one-liner before the function call. I may consider adding this later but no promises. Pull requests are also welcome.
